early_access/master.adoc

```diff
@@ -19,8 +19,36 @@ For {productname} documentation, you should know that:
 * Prior to version 2.9.2, the product was referred to as Quay Enterprise
 endif::downstream[]
 
+include::modules/rn_3_80.adoc[leveloffset=+1]
+
+////
+[id="quay-config-fields-38"]
+== Configuration field updates 
+////
+
+include::modules/config-updates-38.adoc[New configuration fields]
+
+////
+[id='quay-release-notes']
+== {productname} 3.7 Release Notes
+
+{productname} is regularly released, containing new features, bug fixes, and software updates.
+We highly recommend deploying the latest version of {productname}.
+
+ifdef::downstream[]
+For {productname} documentation, you should know that:
```

Comment:
I would recommend omitting that last bit just for clarity. I know that `should` is a "maybe" word in the ISG so you could just remove it entirely. Up to you!

```suggestion
For {productname} documentation:
```

Reply:
Same thing here (just copied it over).